web/api/v1/api_test.go

```diff
@@ -1875,7 +1875,7 @@ func (f *fakeDB) Dir() string {
 }
 func (f *fakeDB) Snapshot(dir string, withHead bool) error { return f.err }
 func (f *fakeDB) Head() *tsdb.Head {
-	h, _ := tsdb.NewHead(nil, nil, nil, 1000)
+	h, _ := tsdb.NewHead(nil, nil, nil, 1000, 1<<14)
```

Comment:
This should be replaced after exposing the constant to avoid any confusion about where `1<<14` came from.

Reply:
Fixed.